docs/contributing.md

```diff
@@ -253,3 +254,19 @@ Pydantic has a badge that you can use to show that your project uses Pydantic. Y
 
 <a href="https://pydantic.dev"><img src="https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/pydantic/pydantic/main/docs/badge/v2.json" alt="Pydantic Version 2" style="max-width:100%;"></a>
 ```
+
+## Adding your library as part of Pydantic tests
+
+To be able to identify regressions early during development, Pydantic runs tests on various third-party projects
+using Pydantic. Are you maintaining an open source project and are you heavily relying on Pydantic? If you match
+some of the following criteria:
+
+- The project is actively maintained.
+- Making use of Pydantic internals (e.g. relying on the [`BaseModel`][pydantic.BaseModel] metaclass, typing utilities).
+- The project is popular enough (although small projects can still be included depending on how Pydantic is being used).
```

Comment:
I feel like we should quantify this (some # of stars, downloads)

Reply:
The thing is it really depends on how Pydantic is used. We can have project with 100k stars but if it is only defining simple models (e.g. models for an API client, and types are well contained) then there's no interest in testing it. On the other hand, a project with 200 stars defining complex custom types is more valuable. So it is hard to quantify :/